include/odp/api/spec/classification.h

```diff
@@ -405,6 +405,15 @@ typedef struct odp_bp_param_t {
 	 */
 	odp_threshold_t threshold;
 
+	/**
+	 * PFC priority level
+	 *
+	 * When enabled (#ODP_PKTIO_LINK_PFC_ON), PFC frames are generated when the above
+	 * threshold is exceeded. The generated frames request the receiver to temporary halt
+	 * transmission of traffic on this priority level (0 .. 7).
+	 */
+	uint8_t pfc_level;
+
 } odp_bp_param_t;
```

Comment:
- Plan with PFC with DIRECT mode is to enable classifier API also with DIRECT mode packet input. I'll need to check that there are no road blocks for that. Assuming not, it would be another PR on top of this one. 

- Details of PFC control in packet output side would be added later on top of this PR

Reply:
I checked pktio/classifier API. Packet input in direct mode with classifier seems pretty straight forward. Mainly additions of odp_pktin_queue_t in place of odp_queue_t in CLS API.